centaur/src/main/resources/standardTestCases/biscayne_read_functions_windows_line_endings.test

```diff
@@ -14,9 +14,9 @@ metadata {
   "outputs.biscayne_read_functions_windows_line_endings.map.y": 600
   "outputs.biscayne_read_functions_windows_line_endings.map.z": 700
   "outputs.biscayne_read_functions_windows_line_endings.map.x": 500
-  "outputs.biscayne_read_functions_windows_line_endings.tsv.0": "[\"line1\",\"line one\"]"
-  "outputs.biscayne_read_functions_windows_line_endings.tsv.1": "[\"line2\",\"line two\"]"
-  "outputs.biscayne_read_functions_windows_line_endings.tsv.2": "[\"line3\",\"line three\"]"
```

Comment:
This is super nice, I've definitely been annoyed in the past about having to craft just the right escape sequence for a test.